src/theme/ui_styles.rs

```diff
@@ -194,12 +194,12 @@ impl UiStyles {
             "nk" => self.size.number_kilo         = pair.to_style(),
             "nm" => self.size.number_mega         = pair.to_style(),
             "ng" => self.size.number_giga         = pair.to_style(),
-            "nh" => self.size.number_huge         = pair.to_style(),
+            "nt" => self.size.number_huge         = pair.to_style(),
```

Comment:
why not keep `nh`, which I believe means «number huge», and changing the man page instead? Same thing for `uh`.

Reply:
If you look at the man page the "t" versions make more logical sense.  Just because we call it huge internally should we call it that externally?  I figured it would be better to pull the bandaid now and break backwards compatibility now and match the docs since most people would no know about it.